include/clink/kernels/opdefs/clink_kernels.td

```diff
@@ -76,4 +90,54 @@ def SquareF64Op: Clink_Op<"square.f64"> {
   let verifier = ?;
 }
 
+def OneHotEncoderLoadOp: Clink_Op<"feature.onehotencoder.load"> {
+  let summary = "feature.onehotencoder.load operation";
+  let description = [{
+     An operation that loads a OneHotEncoderModel from a given path. The path should be a directory
+     containing params and model data saved through
+     org.clink.feature.onehotencoder.ClinkOneHotEnoderModel::save(...).
+
+     Example:
+       %1 = clink.feature.onehotencoder.load %0
```

Comment:
The naming convention of kernels in tfrt is: 
`{dialect_name}.`{function_name}`.`{parameter}`

- dialect_name captures the categorical information, for example `tfrt_test` and `tfrt_data`.
- function_name captures basic logics, e.g. `add` and `multiply`.
- parameter captures information like the parameter type supported by the function, e.g. `i32` and `i64`.

Following this convention, maybe we should name this kernel as either `clink.onehotencoder_load` or `clink_feature.onehotencoder.load`. The latter requires us to add a new dialect named `clink_feature`.

It seems simpler to just use `clink.onehotencoder_load` for now.

Same for other kernel names.

Reply:
OK. I'll make the change.

While one hot encoder might only be used in feature engineering, other algorithms, like decision tree, might be used in multiple use cases, like in classification and regression. In this case `clink.decisiontree_load` might not be enough, and we might need `clink.classification_decisiontree_load` and `clink.regression_decisiontree_load` at that time. 

For now I agree that `clink.onehotencoder_load` is enough.